readme.md

```diff
@@ -52,6 +52,23 @@ module.exports = {
 }
 ```
 
+#### includeAliases
```

Comment:
Add a comment before the docs for `excludeAliases` and `includeAliases` that declares them mutually exclusive.